hack/testing/performance-thresholds.sh

```
@@ -0,0 +1,133 @@
+print_info() { echo -e "${BLUE}ℹ${NC} $*"; }
+print_success() { echo -e "${GREEN}✓${NC} $*"; }
+print_error() { echo -e "${RED}✗${NC} $*" >&2; }
+
+# Parse arguments
+COMMAND="${1:-help}"
+DRY_RUN="${2:-false}"
+
+case "$COMMAND" in
+  collect)
+    print_info "Collecting performance test data..."
+    read -p "How many times to run tests? (recommended: 5-10) " count
```

Shellcheck (0.11.0)

[info] read without -r will mangle backslashes.

(SC2162)